spec/integration/an_organizer_with_interactor_with_after_callbacks_deferred_spec.rb

```diff
@@ -147,5 +150,31 @@ def perform
         ]
       )}
     end
+
+    context 'when interactor is called via organizer' do
+      context 'and interactor is called individually prior' do
```

Comment:
Is there a sad path spec that we can have here? Ie if an interactor fails prior to calling the organizer?

Reply:
Good call...adding that now.

Reply:
Okay, pushed up...let me know what ya think.